collect_app/src/main/java/org/odk/collect/android/application/initialization/ApplicationInitializer.java

```diff
@@ -95,11 +82,10 @@ private void initializeFrameworks() {
     }
 
     private void initializeAnalytics() {
-        boolean isAnalyticsEnabled = generalSettings.getBoolean(GeneralKeys.KEY_ANALYTICS);
-        analytics.setAnalyticsCollectionEnabled(isAnalyticsEnabled);
+        analytics.setAnalyticsCollectionEnabled(false);
 
-        FormUpdateMode formUpdateMode = getFormUpdateMode(context, generalSettings);
-        analytics.setUserProperty("FormUpdateMode", formUpdateMode.getValue(context));
+//        FormUpdateMode formUpdateMode = getFormUpdateMode(context, generalSettings);
```

Comment:
I've created an issue for this as well: https://github.com/getodk/collect/issues/4605